setup.cfg

```diff
@@ -27,6 +27,7 @@ markers =
     large
     serial
     mujoco
+    timeout
```

Comment:
what's the purpose of this?

Reply:
When I run pytest locally, it complained `'timeout' not found in `markers` configuration option`. Not sure why it didn't happen in Travis. Will remove that before merging.